travharv/store.py

```diff
@@ -118,7 +67,7 @@ def insert_for_config(self, graph: Graph, name_config: str) -> None:
         :type name_config: str
         :rtype: None
         """
-        ng: str = self._nmapper.cfgname_to_ng(name_config)
+        ng: str = self._nmapper.key_to_ng(name_config)
```

Comment:
as mentioned, no need to do this yourself anymore, just reuse what is already in the rdf_store ?

https://github.com/vliz-be-opsci/py-RDF-store/blob/d4dddf1777295688899b990b4e437a6d6622ad4f/pyrdfstore/store.py#L126